wicket-util/src/main/java/org/apache/wicket/util/collections/ArrayListStack.java

```diff
@@ -1,139 +0,0 @@
-/*
```

Comment:
I'd prefer if this class is just marked with `@Deprecated` in Wicket 8.x and be removed with Wicket 9.x.